shim/src/handle_system_kvm_create_vm.c

```diff
@@ -25,17 +25,37 @@
  */
 
 #include <debug.h>
+#include <g_hndl.h>
+#include <mv_hypercall.h>
+#include <platform.h>
+#include <shim_vm_t.h>
 #include <types.h>
 
+/* Remove me */
+static uint16_t
+mv_vm_op_create_vm(uint64_t const g_hndl)
+{
+    (void)g_hndl;
+    return 1;
+}
+
 /**
  * <!-- description -->
- *   @brief Handles the execution of kvm_check_extension.
+ *   @brief Handles the execution of kvm_create_vm.
  *
  * <!-- inputs/outputs -->
  *   @return SHIM_SUCCESS on success, SHIM_FAILURE on failure.
  */
 int64_t
-handle_system_kvm_create_vm(void)
+handle_system_kvm_create_vm(struct shim_vm_t *vm)
```

Comment:
the vm object should be const like:
```
struct shim_vm_t *const vm)
```

This states that the stuff that VM points to is mutable, but the VM pointer itself is not. Clang Tidy once turned on will require that you name this variable "pmut_vm" because it states that that the pointee is mutable, but the pointer is const. But for now, the above is fine until we get Clang Tidy turned on.

Reply:
Thanks ..updated const